artifacts_test.py

```diff
@@ -495,20 +489,19 @@ def test_scylla_service(self):  # noqa: PLR0915
             with self.logged_subtest("Check docker latest tags"):
                 self.verify_docker_latest_match_release()
 
+        if self.params.get("run_scylla_doctor"):
```

Comment:
any reason it was moved around ?, like why ?

Reply:
I think it should be ran last as we do some setup afterwards which scylla-doctor might care about, like `perftune`

Reply:
o.k.